src/OpenDB/test/data/gcd/gcd_nangate45_route.def

```diff
@@ -0,0 +1,7410 @@
+    - _683_ DFF_X1 + PLACED ( 44840 92400 ) FS ;
+    - _684_ DFF_X2 + PLACED ( 46360 81200 ) FS ;
+    - _685_ DFF_X1 + PLACED ( 69160 89600 ) N ;
+    - _686_ DFF_X1 + PLACED ( 46740 112000 ) N ;
+    - _687_ DFF_X1 + PLACED ( 71060 112000 ) N ;
+    - _688_ DFF_X1 + PLACED ( 84740 128800 ) N ;
+    - _689_ DFF_X1 + PLACED ( 112480 100800 ) N ;
+    - _690_ DFF_X2 + PLACED ( 130720 120400 ) FS ;
+    - _691_ DFF_X2 + PLACED ( 125780 134400 ) N ;
+    - _692_ DFF_X1 + PLACED ( 66500 131600 ) FS ;
+    - _693_ DFF_X2 + PLACED ( 117040 84000 ) N ;
+    - _694_ DFF_X2 + PLACED ( 104120 47600 ) FS ;
+    - _695_ DFF_X1 + PLACED ( 128060 75600 ) FS ;
+    - _696_ DFF_X1 + PLACED ( 118560 44800 ) N ;
+    - _697_ DFF_X2 + PLACED ( 84360 78400 ) N ;
+    - _698_ DFF_X1 + PLACED ( 87020 44800 ) N ;
+    - _699_ DFF_X1 + PLACED ( 74860 28000 ) N ;
+    - _700_ DFF_X1 + PLACED ( 44840 47600 ) FS ;
+    - _701_ DFF_X1 + PLACED ( 59660 89600 ) N ;
+    - _702_ DFF_X2 + PLACED ( 51300 123200 ) N ;
+    - _703_ DFF_X1 + PLACED ( 78280 98000 ) FS ;
+    - _704_ DFF_X1 + PLACED ( 86640 137200 ) FS ;
+    - _705_ DFF_X2 + PLACED ( 125780 95200 ) N ;
+    - _706_ DFF_X2 + PLACED ( 129580 106400 ) N ;
+    - _707_ DFF_X2 + PLACED ( 119320 140000 ) N ;
+    - _708_ DFF_X2 + PLACED ( 60420 134400 ) N ;
+    - _709_ DFF_X2 + PLACED ( 94240 89600 ) N ;
+    - _710_ DFF_X2 + PLACED ( 107160 84000 ) N ;
+    - _711_ DFF_X2 + PLACED ( 132240 70000 ) FS ;
+    - _712_ DFF_X1 + PLACED ( 121600 30800 ) FS ;
+    - _713_ DFF_X1 + PLACED ( 68780 75600 ) FS ;
+    - _714_ DFF_X1 + PLACED ( 89300 30800 ) FS ;
+    - _715_ DFF_X2 + PLACED ( 62700 28000 ) N ;
+    - _716_ DFF_X1 + PLACED ( 46360 30800 ) FS ;
+    - clkbuf_0_clk BUF_X4 + PLACED ( 89680 84000 ) N ;
+    - clkbuf_2_0__f_clk BUF_X4 + PLACED ( 69920 56000 ) N ;
+    - clkbuf_2_1__f_clk BUF_X4 + PLACED ( 98040 56000 ) N ;
+    - clkbuf_2_2__f_clk BUF_X4 + PLACED ( 67640 112000 ) N ;
+    - clkbuf_2_3__f_clk BUF_X4 + PLACED ( 98040 98000 ) FS ;
+    - rebuffer22 BUF_X1 + PLACED ( 101080 84000 ) N ;
+    - rebuffer23 BUF_X1 + PLACED ( 104120 72800 ) N ;
+    - rebuffer24 BUF_X1 + PLACED ( 106020 72800 ) N ;
+    - rebuffer25 BUF_X4 + PLACED ( 99560 64400 ) FS ;
+    - rebuffer26 BUF_X1 + PLACED ( 104880 109200 ) FS ;
+    - rebuffer27 BUF_X1 + PLACED ( 101840 106400 ) N ;
+    - rebuffer28 BUF_X1 + PLACED ( 88920 67200 ) N ;
+    - rebuffer29 BUF_X2 + PLACED ( 75620 78400 ) N ;
+    - rebuffer31 BUF_X1 + PLACED ( 55100 44800 ) N ;
+    - rebuffer32 BUF_X1 + PLACED ( 58140 72800 ) N ;
+    - rebuffer33 BUF_X4 + PLACED ( 77900 78400 ) N ;
+    - rebuffer34 BUF_X1 + PLACED ( 116280 117600 ) N ;
+    - repeater19 BUF_X4 + PLACED ( 57000 78400 ) N ;
+    - repeater20 BUF_X4 + PLACED ( 41040 70000 ) FS ;
+    - repeater21 BUF_X8 + PLACED ( 44460 70000 ) FS ;
+    - split30 BUF_X2 + PLACED ( 55860 72800 ) N ;
+END COMPONENTS
+PINS 54 ;
+    - clk + NET clk + DIRECTION INPUT + USE SIGNAL
```

Comment:
@rovinski It looks like the clock is marked as signal net. This was generated from the openroad flow tests.

Reply:
It sounds like this needs to be fixed at a higher level then (not for this PR).
But we definitely need to make sure the test case has power pins exposed. This is a key part of LEF abstraction.

Reply:
I agree on the power pins